views/apps/new.html

```diff
@@ -1,241 +1,279 @@
 {{define "body"}}
 
-<div
-  class="w-full lg:w-8/12 mx-auto bg-white rounded-md shadow px-4 lg:px-12 py-4 lg:py-12 mb-10 dark:bg-surface-02dp"
->
-  <h2 class="font-bold text-2xl font-headline mb-4 dark:text-white">
+<div class="mx-auto bg-white rounded-md shadow p-4 md:px-8 md:py-6 mb-10 dark:bg-surface-02dp">
```

Comment:
I think these classes should always be the same... potentially ideally even part of the layout if possible. 

when each page is different it becomes hard to maintain and it becomes hard to change